mlir/utils/jenkins/MIOpenDriver.py

```diff
@@ -243,66 +244,33 @@ def benchmarkMIOpen(commandLine, outputFile, xdlops):
     else:
         # skip the test for non-supported layouts.
         # MIOpenDriver currently only support NCHW.
-        nanoSeconds = 0
-    output(config.generateCSVContent(nanoSeconds), outputFile)
-
-def geo_mean(data):
-    prod = 1
-    n = len(data)
-    for i in data:
-       if i == ',0.0' or i == ',-1.0':
-           # ignore invalid data
-           n = n - 1
-       else:
-           prod = prod * float(i[1:])
-    geoMean = prod ** (1/n)
-    return round(geoMean, ROUND_DIGITS)
-
-def printPerformance(mlirFileName, miopenFileName):
-    with open(mlirFileName, 'r') as mlirOutput:
-        mlirResults = [line[:-1] for line in mlirOutput.readlines()]
-        mlirFlops = [','+row.rstrip().split(',')[-1] for row in mlirResults]
- 
-    with open(miopenFileName, 'r') as miopenOutput:
-        miopenFlops = [','+line.rstrip().split(',')[-1] for line in miopenOutput.readlines()]
+        nanoSeconds = np.nan
+    return config.tableEntry(nanoSeconds)
 
-    results = [ i+j for i, j in zip(mlirResults, miopenFlops)]
-    header = ConvConfiguration.generateCSVHeader()+',MIOpenTFlops'
-    
-    table = PrettyTable()
-    table.field_names = header.split(',')
-    for res in results:
-        table.add_row(res.split(','))
+def printPerformance(mlir_df, miopen_df):
+    df = mlir_df.merge(miopen_df, on=ConvConfiguration.TABLE_COLUMNS[:-1],
+                           suffixes=('', ' (MIOpen)'))
+    df.rename(columns={'TFlops (MIOpen)': 'MIOpen TFlops'}, inplace=True)
 
-    # Add geometric means to the table
-    mlirGeoMean = geo_mean(mlirFlops)
-    miopenGeoMean = geo_mean(miopenFlops)
+    df['Speedup'] = df['TFlops'] / df['MIOpen TFlops']
+    df.to_csv(reportUtils.PERF_REPORT_FILE, index=False)
 
-    means = [' ']*len(header.split(',')[0:-2])
-    means.append(str(mlirGeoMean))
-    means.append(str(miopenGeoMean))
-    means[0] = 'Geo Mean';
-    table.add_row(means)
+    COLUMNS_TO_AVERAGE = ['TFlops', 'MIOpen TFlops', 'Speedup']
```

Comment:
I'd suggest to change them as 'MLIR TFlops', 'MIOpen TFlops', 'MLIR/MIOpen'. I always wonder how speedups are computed when I read a report like this.